ruby/ql/test/library-tests/dataflow/summaries/summaries.rb

```diff
@@ -47,3 +47,20 @@ def userDefinedFunction(x, y)
 Foo.blockArg do |x|
   sink(x.preserveTaint("taint"))
 end
+
+sink(Foo.namedArg(foo: tainted))
+sink(Foo.namedArg(tainted))
+
+Foo.intoNamedCallback(tainted, foo: ->(x) {
+  sink(x)
+})
+Foo.intoNamedParameter(tainted, ->(foo:) {
+  sink(foo)
```

Comment:
We should document that flow is missing in this case

Reply:
Dang, I missed that. I'll try to introduce inline assertions to this test.

Reply:
Updated to use inline test expectations, much better overall, and it's now clear that this flow is missing.

I haven't investigated why the flow is missing, though. I was under the impression that providing the correct `ArgumentPosition` and `ParameterPosition` values would be enough.

Reply:
@hvitved would you like it addressed in this PR? Please let me know either way

Reply:
No, let's do that follow-up